pkg/common/config/config.go

```diff
@@ -250,12 +250,17 @@ var Addons = struct {
 	// RunCleanup is a boolean to specify whether the testHarnesses should have a separate
 	// cleanup phase. This phase would run at the end of all e2e testing
 	RunCleanup string
+
+	// CleanupHarnesses is a comma separated list of container images that will clean up any
+	// artifacts created after test harnesses have run
+	CleanupHarnesses string
 }{
-	IDsAtCreation: "addons.idsAtCreation",
-	IDs:           "ids",
-	TestHarnesses: "testHarnesses",
-	TestUser:      "addons.testUser",
-	RunCleanup:    "addons.runCleanup",
+	IDsAtCreation:    "addons.idsAtCreation",
+	IDs:              "ids",
+	TestHarnesses:    "testHarnesses",
+	TestUser:         "addons.testUser",
+	RunCleanup:       "addons.runCleanup",
+	CleanupHarnesses: "cleanupHarnesses",
```

Comment:
Huh, odd. I have a nit to expand all of these to addons.*, so:

```
addons.ids
addons.testHarnesses
addons.cleanupHarnesses
```

I must have forgotten this when I set this up. I realize that ids/testHarnesses isn't part of your change, but mind fixing them while we're here?